argus/client/windows.py

```diff
@@ -24,6 +24,8 @@
     import io as StringIO
```

Comment:
for the commit message:

1. no need for upper casing
2. an upper*
3. Commit message: If a winrm command is sent to the Windows client machine, it can hang for an unlimited amount of time. This commit adds the option to set a configurable timeout for the winrm calls.